ts/Series/Sankey/SankeySeries.ts

```diff
@@ -1258,16 +1269,50 @@ export default SankeySeries;
  * @type {string}
  * @product highcharts
  *//**
+ * This option is deprecated, use
+ * {@link Highcharts.SankeyNodeObject#offsetHorizontal} and
+ * {@link Highcharts.SankeyNodeObject#offsetVertical}  instead.
+ *
  * The vertical offset of a node in terms of weight. Positive values shift the
  * node downwards, negative shift it upwards.
  *
+ * If a percantage string is given, the node is offset by the percentage of the
+ * node size plus `nodePadding`.
+ *
  * @see {@link https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/plotoptions/sankey-node-column/|Highcharts-Demo:}
  *         Specified node offset
  *
+ * @deprecated
  * @name Highcharts.SankeyNodeObject#offset
- * @type {number}
+ * @type {number|string}
  * @default 0
  * @since 6.0.5
+ *//**
+ * The horizontal offset of a node. Positive values shift the node right,
+ * negative shift it left.
+ *
+ * If a percantage string is given, the node is offset by the percentage of the
+ * node size.
+ *
+ * @see {@link https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/plotoptions/sankey-node-column/|Highcharts-Demo:}
+ *         Specified node offset
+ *
+ * @name Highcharts.SankeyNodeObject#offsetHorizontal
+ * @type {number|string}
+ * @since next
+ *//**
+ * The vertical offset of a node. Positive values shift the node down,
+ * negative shift it up.
+ *
+ * If a percantage string is given, the node is offset by the percentage of the
+ * node size.
+ *
+ * @see {@link https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/plotoptions/sankey-node-column/|Highcharts-Demo:}
+ *         Specified node offset
+ *
+ * @name Highcharts.SankeyNodeObject#offsetVertical
```

Comment:
What about naming it `horizontalOffset` and `verticalOffset`?

Idea: instead of deprecating `offset`, can we use the old `offset` as `horizontalOffset`, and add only `verticalOffset` as a new option?

Reply:
The problem with keeping the old one as one of the new ones is if that the old `offset` is either horizontal or vertical depending on the chart being inverted or not inverted, so it won't be very intuitive IMO.

I like the current names as they get to be placed near the `offset` when sorted alphabetically 😄 
Why different names?